api/public/schema/projectSchema.js

```diff
@@ -38,6 +38,10 @@ const schema = {
       type: "string",
       required: false
     },
+    followers: {
```

Comment:
shouldn't this be an array of strings?

Reply:
wait that's weird it still works with my following backend stuff